index.html

```diff
@@ -37,6 +37,48 @@
     </nav>
 
 
+    <main>
+        <div class="theme">
+            <div class="flex">
+                <div>
+                    <h1 class="h1"> Meow in! </h1>
+                    <p> You know that feeling when you miss sunshine?
+                        Yep, <em> that </em> is how much we missed you. <strong> Welcome back</strong>.
+                    </p>
+                </div>
+                <img src="https://www.smashingmagazine.com/images/smashing-cat/singing-barista.svg" alt="logo"
+                    width="250" height="100" class="floatright" />
+            </div>
+            <form class="main-form">
+                <label class="block">
+                    <span class="margin-bottom"> Your email </span>
+                    <input class="login-input" type="text" name="email" required>
+                </label>
+
+                <label>
+                    <span class="margin-bottom margin-top"> Memorable password (yep, that one) </span>
```

Comment:
Same as above. The `margin-bottom`, `margin-top` classes are confusing because they don't specify the amount.